tools/templates/files/fileTemplates/Template Presentation Classes.kt.child.2.kt

```diff
@@ -0,0 +1,42 @@
+#if (${PACKAGE_NAME} && ${PACKAGE_NAME} != "")package ${PACKAGE_NAME}#end
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import io.element.android.libraries.designsystem.preview.ElementPreviewDark
+import io.element.android.libraries.designsystem.preview.ElementPreviewLight
+import io.element.android.libraries.designsystem.theme.components.Text
+
+@Composable
+fun ${NAME}View(
+    state: ${NAME}State,
+    modifier: Modifier = Modifier,
+) {
+    Box(modifier, contentAlignment = Alignment.Center) {
+        Text(
+            "${NAME} feature view",
+            color = MaterialTheme.colorScheme.primary,
+        )
+    }
+}
+
+@Preview
+@Composable
```

Comment:
It could be a good time to unify these using `@DayNightPreviews` maybe?

Reply:
I just realized that: a65eb36